docs/microservices.json

```diff
@@ -25,8 +25,8 @@
             "colour": "#f39c12"
         },
         {
-            "name": "CMC",
-            "colour": "#f30000"
```

Comment:
why was CMC removed?

Reply:
Sorry for the confusion.Removed in error .Will close this one and create a new one.